charts/gitops-runtime/Chart.yaml

```diff
@@ -15,10 +15,8 @@ annotations:
   artifacthub.io/alternativeName: "codefresh-gitops-runtime"
   artifacthub.io/prerelease: "true"
   artifacthub.io/changes: |
-    - kind: changed
-      description: Updated `app-proxy` to `1.2268.0`
-    - kind: changed
-      description: updated default value of workflow-reporter SA to "codefresh-sa"
+    - kind: feat
```

Comment:
Not a valid ArtifactHub change type 
supported kinds are added, changed, deprecated, removed, fixed and security.
https://artifacthub.io/docs/topics/annotations/helm/